src/internal/do-not-use/chart-tooltip.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import InternalChartTooltip, { ChartPopoverProps } from '../components/chart-popover/index.js';
+export type InternalChartTooltipProps = ChartPopoverProps;
+export { InternalChartTooltip };
```

Comment:
Why not use default exports for these components?

Reply:
That is to reinforce the component name, which includes the word "internal".